ep0099/doc.go

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018 The Periph Authors. All rights reserved.
+// Use of this source code is governed under the Apache License, Version 2.0
+// that caan be found in the LICENSE file.
+
+// Package ep0099
+//
+// Datasheet
+// https://wiki.52pi.com/index.php/DockerPi_4_Channel_Relay_SKU:_EP-0099
+//
+// Examples
```

Comment:
Remove, not relevant for users not implementers.

Reply:
looks like all other devices have links to data sheets in doc.go as I was trying to be consistent.

Reply:
I was commenting on the Examples, not Datasheet.